src/api/app/models/user.rb

```diff
@@ -911,6 +911,14 @@ def update_notifications(params)
     User.update_notifications(params, self)
   end
 
+  def combined_rss_feed_items
+    feed_items = rss_feed_items.to_a
```

Comment:
The second way doesn't work because the default scope of the order... and if I unscope it then it just will return the group related ones...

I will go for the first solution